### PR TITLE
Corrected base64 decode switch

### PR DESCRIPTION
**Pull request by aggress**
Confirmed on CentOS and Ubuntu

### CODE REVIEW

**Comment by aggress** [comment]
Thought so. This can't move forwards as is without checking for OS type. -D is correct for darwin but not for Linux. I will add that check.

File: docker/local-universe/add-stub-universe.sh [hunk 1]
```diff
@@ -47,14 +54,14 @@ jq --arg PACKAGE "$PACKAGE" '.packages[] | select(.name == $PACKAGE)' $FILE > st
...
+cat stub-repo/$PACKAGE.json | jq -r '.marathon.v2AppMustacheTemplate' | ${BASE64} > stub-repo/packages/${LETTER}/${PACKAGE}/0/marathon.json.mustache
 
 done
 
 echo ""
 echo "Full stub-repo contents:"
 ls -alh stub-repo/packages/*/*/*
 
-echo ""
+echo ""
```
Comment (jsancio) [inline docker/local-universe/add-stub-universe.sh]: Add a newline.

Comment (aggress) [reply docker/local-universe/add-stub-universe.sh]: He's on 🌴  so may not get to see this for a bit

Comment (jsancio) [reply docker/local-universe/add-stub-universe.sh]: Sounds good. Merge it when you are ready.

**Pull request merged**